languages/fsharp/exercises/concept/values/.meta/design.md

```diff
@@ -0,0 +1,61 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student the basics of the Concept of Values in [F#][values].
+
+## Learning objectives
+
+- Know what a value is.
+- Know how to define a binding.
+- Know how to define a function.
```

Comment:
Well, I originally envisioned the exercise to just cover bindings, but have since then decided to also include functions as well (and exclude mutability). I've updated the `introduction.md` already, but the rest of the exercise also needs to be updated. Sorry that I didn't make that clear.